tools/FakeItEasy.Build/Program.cs

```diff
@@ -65,11 +65,29 @@
 Target(
     "check-docs-links",
     DependsOn("generate-docs"),
-    () => Run("linkchecker", "--ignore-url=sitemap.xml --ignore-url=404.html --check-extern -F html/utf-8/artifacts/docs-link-check.html ./artifacts/docs/index.html"));
+    () => Run("uv", "run linkchecker --ignore-url=sitemap.xml --ignore-url=404.html --check-extern -F html/utf-8/artifacts/docs-link-check.html ./artifacts/docs/index.html"));
 
 Target(
     "generate-docs",
-    () => Run("python", "-m mkdocs build --clean --site-dir artifacts/docs --config-file mkdocs.yml --strict"));
+    DependsOn("create-python-version-file"),
+    () => Run("uv", "run mkdocs build --clean --site-dir artifacts/docs --config-file mkdocs.yml --strict"));
+
+// uv really likes there to be a .python-version file to specify which version of python it should use.
+// Rather than maintain a duplicate of the version in pyproject.toml, we'll generate the .python-version file from the latter.
+Target(
+    "create-python-version-file",
+    () =>
+    {
+        // expect the line to look something like like
+        // requires-python = "~=x.yz"
+        // with no surrounding whitespace. We want the x.yz part.
+        var prefix = "requires-python = \"~=";
+        var pythonVersion = File.ReadAllLines("pyproject.toml")

```

Comment:
nit: you can use `File.ReadLines` instead of `File.ReadAllLines` (returns a lazy `IEnumerable<string>` instead of an array)